test/tests/styleTest.js

```diff
@@ -0,0 +1,41 @@
+"use strict";
+
+describe("Zotero.Styles", function() {
+	var styleID = "http://www.zotero.org/styles/cell";
+	var styleFile = getTestDataDirectory();
+	var style;
+	styleFile.append('cell.csl');
+	
+	before(function* () {
+		yield Zotero.Styles.init();
+		style = yield Zotero.File.getContentsAsync(styleFile);
+	});
+	
+	describe("Zotero.Styles.install", function() {
+		afterEach(function* (){
+			assert.isOk(Zotero.Styles.get(styleID));
+			yield Zotero.Styles.get(styleID).remove();
+		});
+		
+		it("Installs the style from string", function* () {
```

Comment:
> Sorry, I know you dislike it, but that's the Mocha convention we've used for everything.

It's a bit ugly, but I strongly prefer consistency with some ugliness over ideological chaos. I simply forget that we do "should" everywhere, especially when in a new testfile, so do continue pointing it out in the future.